src/services/conseillers/conseillers.class.js

```diff
@@ -1242,6 +1230,10 @@ exports.Conseillers = class Conseillers extends Service {
           conseiller.updatedAt = updatedAt;
           conseiller.codeCom = codeCom;
 
+          await db.collection('misesEnRelation').deleteMany({
+            'conseiller.$id': conseiller._id,
+            'statut': { '$nin': ['finalisee', 'finalisee_rupture'] } });
```

Comment:
plutot `$in: ['finalisee_non_disponible`, 'non_disponible']` ?!
car il y aura d'autre statut lié au reconventionnement etc...

Reply:
avec le statut nouvelle aussi ?

Reply:
Non : 
Ici on conserve les contrats finalisés car on veut garder l'historique du conseiller.
Il y aura recréation des mises en relation par le script parce qu'on change de distance, et que les structures ne doivent plus voir le conseiller.
Pour le reconventionnement, quel statut peut poser problème ?

Reply:
Pour moi les statuts intéressé, pré-selectionné etc n'ont pas lieu d'être si le conseiller s'en va ailleurs.

Reply:
je suis d'accord avec toi pour le interessé 👍 
- le validé(recrutee) on le conserve  !?
- nonInteressee faut le garder je pense, car il devrait plus etre visible par la SA qui met ce statut
Pour le reconventionnement , il y aura : 
- terminee
- renouvellement_initiee

Reply:
En fait je suis en train de me demander : ce delete est à faire que dans le chg de zone géo ou chag de date / disponibilité ?

Reply:
hum c'est vrai qu'on pourrait se contenté de la zone géo je pense